flake8_trio/base.py

```diff
@@ -2,7 +2,24 @@
 
 from __future__ import annotations
 
-from typing import Any, NamedTuple
+from dataclasses import dataclass
+from typing import TYPE_CHECKING, Any, NamedTuple
+
+if TYPE_CHECKING:
+    from collections.abc import Collection
+
+
+@dataclass
+class Options:
+    # enable and disable have been expanded to contain full codes, and have no overlap
```

Comment:
not sure having both enable and disable is any upside. And if anything should have a enabled_or_autofixed ?

Reply:
ye I realized as I wrote my comment that it was bad design, also renamed the variables and improved comments.